vendor/moko/lw001-bg.yaml

```diff
@@ -1,19 +1,32 @@
-name: LW001-BG
-description: The Moko Smart LW001-BG is a LoRaWAN® indoor and outdoor tracker that consists of GPS, accelerometer, temperature, humidity, and pressure sensors. Ideal to optimize the logistic process, asset management, warehouse and inventory scenarios, outdoor tracking, theft protection, and much more.
+name: LW001-BG PRO
+description: Ultra-low power consumption, indoor&outdoor(BLE&WIFI&GPS) LoRaWAN Tracker
 
 # Hardware versions (optional, use when you have revisions)
 hardwareVersions:
-  - version: '1.0.1'
+  - version: '2.4'
     numeric: 1
 
 # Firmware versions (at least one is mandatory)
 firmwareVersions:
   - # Firmware version
-    version: '1.0.1'
+    version: '1.0.7'
```

Comment:
do not delete the old firmware version, this can affect the devices already deploy, add the new firmware version below